README.md

```diff
@@ -63,6 +63,12 @@ We recommend using Sparrow unless you're running an instance of Bitcoin Core. Fo
 10) The tabs on the left will turn blue.
 11) Go to the receive tab. You will need this address for .env file configuration.
 
+### Encrypting the wallet seed
+To encrypt the wallet seed run:
+```agsl
+node -r dotenv/config scripts/encrypt-seed.js
```

Comment:
Can we make this an npm command? Also is agsl a typo?

Reply:
good call.
agsl is always auto-recommended by my editor, im not sure what it is lol